traits/observers/tests/test_list_change_event.py

```diff
@@ -0,0 +1,62 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+import unittest
+
+from traits.trait_list_object import TraitList
+from traits.observers._list_change_event import (
```

Comment:
nitpick: :abc:  Yes, I know we're not consistent about import order in existing code, but in new code please keep things in alphabetical order, so that we can slowly move _towards_ consistency.